packages/@o3r/core/README.md

```diff
@@ -81,26 +81,28 @@ Then uncomment the following lines in the `src/styles.scss` file to apply the Ot
-| fixture             | Adding functions to an Otter fixture based on a selector and default methods. | `ng generate @o3r/core:fixture`             |
-| component           | Create a new Otter component in your library / application.                   | `ng generate @o3r/core:component`           |
-| component-container | Create a new Otter container component in your library / application.         | `ng generate @o3r/core:component-container` |
-| component-presenter | Create a new Otter presenter component in your library / application.         | `ng generate @o3r/core:component-presenter` |
-| playwright-scenario | Create a new Playwright scenario in your application.                         | `ng generate @o3r/core:playwright-scenario` |
-| store               | Create a new store in your library / application.                             | `ng generate @o3r/core:store`               |
-| store-entity-async  | Create an entity async new store in your library / application.               | `ng generate @o3r/core:store-entity-async`  |
-| store-entity-sync   | Create an entity sync new store in your library / application.                | `ng generate @o3r/core:store-entity-sync`   |
-| store-simple-async  | Create a simple async new store in your library / application.                | `ng generate @o3r/core:store-simple-async`  |
-| store-simple-sync   | Create a simple sync new store in your library / application.                 | `ng generate @o3r/core:store-simple-sync`   |
-| store-action        | Create an action into an existing store.                                      | `ng generate @o3r/core:store-action`        |
-| storybook-component | Create an Storybook file for a presenter.                                     | `ng generate @o3r/core:storybook-component` |
-| renovate-bot        | Create a basic Renovate Bot                                                   | `ng generate @o3r/core:renovate-bot`        |
-| module              | Create a new Otter compatible module in your monorepo                         | `ng generate @o3r/core:module`              |
-| ng-add-create       | Include `ng add` schematic into your project                                  | `ng generate @o3r/core:ng-add-create`       |
+| Schematics          | Description                                                                   | How to use                 |
+| ------------------- | ----------------------------------------------------------------------------- | -------------------------- |
+| add                 | Include Otter in a library / application.                                     | `ng add @o3r/core`         |
+| page                | Create a new Page in your application.                                        | `ng g page`                |
+| service             | Create a new Otter Service in your library / application.                     | `ng g service`             |
```

Comment:
is there no risk of conflict with the default Angular generators ?

Reply:
Yes and this is the goal.
Actually `ngCli` is using (and displaying with `ng help`) the generators of the collections listed in angular.json 's [schematicCollections](https://angular.io/guide/workspace-config#angular-cli-configuration-options) config.

The priority is given to the first on of the list in case of conflict and the default Angular generators are provided by de page `@schematics/angular`.
This means that if you do not want to override the default Angular generators you can configure the `angular.json` as following:
```json
{
  "schematicCollections": [
    "@schematics/angular",
    "@o3r/core"
  ]
}
```

In our case we **do want** to override the default Angular schematics with our so we *unshift* the `@o3r/core` collection within this array.

> **note**: The default overriden generators can still be used with it full name: `ng g @schematics/angular:service`